ban/__init__.py

```diff
@@ -0,0 +1,97 @@
+""" ban init file """
+import os
+import json
+import logging
+import traceback
+import requests
+import jsend
+import azure.functions as func
+from requests.models import Response
+from shared_code import common
+
+# map existing ban lookup fieldnames to socrata fieldnames
+ban_map = {
+    "BAN": "ban",
+    "OWNERS": "owners",
+    "BusinessName": "businessname",
+    "DBAName": "dbaname",
+    "StreetAddress": "streetaddress",
+    "City": "city",
+    "State": "state",
+    "PostalCd": "postalcd",
+    "LIN": "lin",
+    "BusstartDate": "busstartdate",
+    "BusEndDate": "busenddate",
+    "LocstartDate": "locstartdate",
+    "LocEndDate": "locenddate",
+    "MailingAddress": "mailingaddress",
+    "MailCityStateZip": "mailcitystatezip",
+    "LocationNumber": "locationnumber",
+    "OrgType": "orgtype",
+    "MailZip": "postalcd",
+    "MailState": "state",
+    "MailCity": "city"
+}
+
+headers = {
+    "Access-Control-Allow-Origin": "*"
+}
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    """ main function for ban """
+    logging.info('Status processed a request.')
+
+    try:
+        common.validate_access(req)
+
+        response = Response()
+        if req.get_body() and len(req.get_body()):
+            response = common.get_http_response_by_status(202)
+
+            req_json = req.get_json()
+            print(f"ban req_json: {req_json}")
+
+            params = {}
+            if "BAN" not in req_json:
+                raise ValueError("Missing BAN parameter")
+
+            # query socrata
+            params["BAN"] = req_json.get("BAN")
+            response = requests.get(
+                os.getenv('BAN_API_URL'),
+                headers={
+                    'Authorization': 'Basic ' + os.getenv('ADDRESS_SVC_AUTH_API_KEY'),
+                    'X-App-Token': os.getenv('ADDRESS_SVC_APP_TOKEN')
```

Comment:
Probably rename `ADDRESS_SVC_AUTH_API_KEY` to something like `BAN_API_AUTH_API_KEY`, and similarly `ADDRESS_SVC_APP_TOKEN` to `BAN_API_APP_TOKEN`

Reply:
7ab99d1